### PR TITLE
add NASA tab to basemaps modal

### PR DESCRIPTION
**Pull request by matallo**
- replicated nasa tab behaviour
- added a new datepicker (non range) component
- fixed an issue detected in sibling tabs with OK button, and not needed `modalFooter` options

nasabasemaps

available in ded02

CR @xavijam

### CODE REVIEW

**Comment by matallo** [comment]
hound changes made @xavijam

**Comment by xavijam** [comment]
Review my comments please :)

File: lib/assets/javascripts/cartodb3/components/modals/add-basemap/wms/enter-url.tpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="u-flex u-alignCenter Modal-basemapContainer">
   <h3 class="CDB-Text CDB-Size-large u-mainTextColor Modal-titleBasemap"><%- _t('components.modals.add-basemap.wms.insert') %></h3>
-  <div class="CDB-Text u-flex u-alignCenter js-name-field">
```
Comment (matallo) [inline lib/assets/javascripts/cartodb3/components/modals/add-basemap/wms/enter-url.tpl]: missed when replicating new modals, not needed in this view nor subsequent 👍

**Comment by matallo** [comment]
🚀

**Pull request merged**